nzshm_common/grids/region_grid.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+from nzshm_common.grids.nz_0_2_nb_1 import NZ_0_2_nb_1
+from nzshm_common.grids.wlg_0_01_nb_1 import WLG_0_01_nb_1
+from nzshm_common.grids.wlg_0_05_nb_1 import WLG_0_05_nb_1
+
+
+class RegionGrid(Enum):
+    NZ_0_2_NB_1 = ('NZ', 0.2, 1, NZ_0_2_nb_1())
```

Comment:
OK as discussed make these using named args pls (for readabliity) and add version - thks!